### PR TITLE
Add vide@0.5.0 w/ npm auto-update

### LINKED ISSUE #8994: [Request] Add jquery.vide
**Library name:** Vide
**Git repository url:** https://github.com/VodkaBears/Vide
**npm package url(optional):** 
**License(s):**
**Official homepage:** https://github.com/VodkaBears/Vide
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by Piicksarn**
@Amomo, @maruilian11,
i need ur help for checking the pr, thx.
 close #8994
--
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required):  https://github.com/VodkaBears/Vide
 * Official website (optional, not the repository): http://vodkabears.github.io/vide/
 * NPM package url (optional): https://www.npmjs.com/package/vide
 * GitHub / Bitbucket popularity (required): 
   - Count of stars:  2947
   - Count of watchers:  106
   - Count of forks:  402
 * NPM download stats (optional):
   - Downloads in the last day:  16
   - Downloads in the last week:  84
   - Downloads in the last month:  288

# Essential checklist
 * [ ] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by maruilian11** [comment]
you should not ping the author in comment, it's already in commit msg.
mention the issue number in beginning.

**Comment by maruilian11** [comment]
there are only 10 versions in npm package, please add other versions from git repo.

File: ajax/libs/vide/package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    "jquery-plugin",
+    "jquery",
+    "plugin",
+    "video",
+    "background",
+    "ui",
+    "responsive",
+    "declarative"
+  ],
+  "homepage": "http://vodkabears.github.io/vide/",
```
Comment (maruilian11) [inline ajax/libs/vide/package.json]: please use https

**Comment by Piicksarn** [comment]
@Amomo, @maruilian11, i have it modified, thanks.

**Comment by maruilian11** [comment]
you still didn't add versions in git repo.

**Comment by Piicksarn** [comment]
@PeterDaveHello, in this case, may i use git autoupdate in this commit, then modified the autoupdate config part after the old versions be added by robot.

**Comment by PeterDaveHello** [comment]
@Piicksarn you should try to grab them from git auto-update and add them all in this PR.

**Comment by Piicksarn** [comment]
@maruilian11, would you please help me for checkout the pr? thanks!

**Comment by pvnr0082t** [comment]
@Piicksarn I think you should still use npm auto-update config in `package.json`. Just try to use [autoupdate](https://github.com/cdnjs/autoupdate) to fetch some other versions npm can't get from git repo and add them in this PR.

**Comment by Piicksarn** [comment]
@maruilian11, @Amomo 
please help me checkout the pr, thanks!

**Comment by maruilian11** [comment]
you miss license field, and your commit message wrap weirdly. 
"the other" means one thing.

**Comment by Piicksarn** [comment]
@cdnjs/intern @cdnjs/intern2 
I've it rebased to master, would you mind to have a check?

Thanks

**Pull request merged**